test/common/test_runner.go

```diff
@@ -115,6 +119,63 @@ func TearDown(client *Client) {
 	}
 }
 
+// CopySecret will copy a secret from one namespace into another.
+// If a ServiceAccount name is provided then it'll add it as a PullSecret to
+// it.
+// It'll either return a pointer to the new Secret or and error indicating
+// why it couldn't do it.
+func CopySecret(client *Client, srcNS string, srcSecretName string, tgtNS string, svcAccount string) (*corev1.Secret, error) {
+	// Get the Interfaces we need to access the resources in the cluster
+	srcSecI := client.Kube.Kube.CoreV1().Secrets(srcNS)
+	tgtNSSvcAccI := client.Kube.Kube.CoreV1().ServiceAccounts(tgtNS)
+	tgtNSSecI := client.Kube.Kube.CoreV1().Secrets(tgtNS)
+
+	// First try to find the secret we're supposed to copy
+	srcSecret, err := srcSecI.Get(srcSecretName, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	// Just double checking
+	if srcSecret == nil {
+		return nil, goerrs.New("Error copying Secret, it's nil w/o error")
+	}
+
+	// Found the secret, so now make a copy in our new namespace
+	newSecret, err := tgtNSSecI.Create(
+		&corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: srcSecretName,
+			},
+			Data: srcSecret.Data,
+			Type: srcSecret.Type,
+		})
+
+	// If the secret already exists then that's ok - some other test
+	// must have created it
+	if err != nil && !errors.IsAlreadyExists(err) {
+		return nil, fmt.Errorf("Error copying the Secret: %s", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("frror copying the Secret: %s", err)
```
See: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
I think I'll go with `error` instead of `frror`  :-)